dialogue/TownNPCs/Olive/InitialInteractions/Pistachio/Pistachio_Olive_First2.json

```diff
@@ -13,6 +13,12 @@
       "Still! Even if you come back now, there's no guarantee you will next time!",
       "Are you still going despite knowing that?!"
     ],
+    "pl": [
+      "Haa, to jesteś z nimi teraz, Stacju?\nI nie jesteś tu, aby zostać?",
```

Comment:
Also not sure about translating "Stachi" here too

Reply:
I was thinking that the isolation from the English name might make the nickname strange for Polish speakers, but I can change it back if I need to.

Reply:
That's a really tough one. You got the point, but I'm not sure if there's any way to translate it to polish without making it sound kinda weird.

Reply:
Danish leaves the name alone, so if it would be better to leave it as-is then that's fine.